### PR TITLE
Add list of Featured third-party Apps

### PR DESCRIPTION
**Pull request by librarianmage**
Fixes https://github.com/HabitRPG/habitrpg/issues/6954
### Changes

Adds list of featured third-party apps to Settings -> API with a link to the other apps/extensions

---

UUID: 95f16006-1504-4a9f-95b6-0c7314dbd7ee

### CODE REVIEW

File: common/locales/en/settings.json [hunk 1]
```diff
@@ -66,6 +66,10 @@
     "API": "API",
     "APIText": "Copy these for use in third party applications. However, think of your API Token like a password, and do not share it publicly. You may occasionally be asked for your User ID, but never post your API Token where others can see it, including on Github.",
     "APIToken": "API Token (this is a password - see warning above!)",
+    "thirdPartyApps": "Third Party Apps",
+    "beeminder": "Beeminder",
+    "chromeChatExtension": "Chrome Chat Extension",
+    "otherExtensions": "<a target='blank' href='http://habitica.wikia.com/wiki/App_and_Extension_Integrations'>Other Extensions</a>"
```
Comment (crookedneighbor) [inline common/locales/en/settings.json]: Missing a comma at the end of this line.

**Comment by Alys** [comment]
We'll add some descriptive text for each as well. For Beeminder, use this for now (might change later after we and the Beeminder folks see what it looks like):

Let Beeminder automatically monitor your Habitica To-Dos. You can commit to maintaining a target number of To-Dos completed per day or per week, or you can commit to gradually reducing your remaining number of uncompleted To-Dos. (By "commit" Beeminder means under threat of paying actual money! But you may also just like Beeminder's fancy graphs.)

For the chat extension, use this which I took from its wiki page:

The Chrome Chat Extension for Habitica adds an intuitive chat box to all of habitica.com. It allows users to chat in the Tavern, their party, and any guilds they are in.

For the DDT use this unless anyone comes up with something better:

A webpage, that shows you certain information from your Habitica account, such as statistics about your tasks, equipment, and skills.

Once that's in, please show us a screenshot of what it all looks like.

**Comment by librarianmage** [comment]
@Alys I have added the descriptions, but the computer I am using is refusing to run Habitica, so I will try to get you a screenshot ASAP.

**Comment by librarianmage** [comment]
image
@Alys Done!

**Comment by Alys** [comment]
Looking pretty good! Let's put them in alphabetical order though (Beeminder, Chrome Chat, DDT, Other), however don't do that yet because I'll ask the Beeminder folks if they want changes to the text.

**Comment by librarianmage** [comment]
I can reorder them now, and change the text later.

**Comment by Alys** [comment]
@MathWhiz Thank you! Please do reorder. We've heard from Beeminder that they're happy with the text as it is, so we'll go live for this once they're in alpha order.

**Comment by crookedneighbor** [comment]
Any chance we could make this a bootstrap table instead of a list? I think that would look better.

**Comment by librarianmage** [comment]
Sure! Would it be better horizontally or vertically?

**Comment by librarianmage** [comment]
@crookedneighbor What text would you suggest for 'Other'?

**Comment by Alys** [comment]
It's not impossible that we'll add more text later for the Beeminder integration, and future integrations might also need large descriptions, so use whatever table layout will look reasonable when the descriptions are of very different sizes. I'm not sure a standard HTML table would ever look good for that but I'm not familiar with the ways that Bootstrap can apply layout improvements.

For "Other", use something like "Find other apps, extensions, and tools on the Habitica wiki."

**Comment by librarianmage** [comment]
Note: Lots of images!

This is what a standard `.table` looks like:
image
However, I can make it a `.table-striped` table:
image

**Comment by Alys** [comment]
I think I preferred the original design. I'm not sure it really works as tabular data.

If we keep the table layout, in the locales file, extentionName should be extensionName. :)

If we revert to the list layout, we should keep the description for "Other".

**Comment by crookedneighbor** [comment]
Let's just do the list. This is not quite what I had in mind.

**Comment by librarianmage** [comment]
Ok!

**Comment by librarianmage** [comment]
@Alys @crookedneighbor I have reset the changes

**Comment by Alys** [comment]
@MathWhiz I'm sorry, but could you put them into alphabetical order? That should then be the final change!

**Comment by librarianmage** [comment]
Oops, I thought I did

**Comment by librarianmage** [comment]
@Alys @crookedneighbor Sorted!

**Comment by Alys** [comment]
Thank you! I've noted this towards your next tier.

**Pull request merged**